docs/testing.md

```diff
@@ -32,27 +32,80 @@ Those can be ignored and `/skip`ed. In example:
 ```
 Upstream issue: https://github.com/openshift/ci-operator-prowgen/issues/79 
 
+## Verify 
+
+This repository runs multiple checks to **verify** if it complies with standard go conventions.  The scripts from the target run through a series of checks to ensure the repository remains compliant with standard go practices. This can be ran **locally** with the command `make verify`.  When creating a pull-request, this tests is part of the acceptance criteria. 
+
+```makefile
+verify:
+    ./hack/verify/validate-generated.sh
+    go vet ./...
```

Comment:
Should be `go get`?

We might not want to include this snippet here since it will change, perhaps we can instead link to the target in the Makefile?

Reply:
I think it should be `go vet` because it's checking the constructs of the go code written in the directories below.  It will fail if the code written fails to comply with the go vet constructs

Reply:
Oh, okay. I didn't actually read the Makefile, I just thought this was a typo at a glance :) Second half of my comment still applies I think.

Reply:
agree with "We might not want to include this snippet here since it will change" - just remove the snippet